src/core/Distribution.pm

```diff
@@ -57,7 +112,7 @@ class Distribution::Resources does Associative {
     }
 
     method AT-KEY($key) {
-        CompUnit::RepositoryRegistry.repository-for-spec($.repo).resource($.dist-id, $key)
+        CompUnit::RepositoryRegistry.repository-for-spec($.repo).resource($.dist-id, "resources/$key")
```

Comment:
Certainly worth fixing. But if this upgrade breaks already installed modules, we need to include the change in the upgrade to the repository layout v1 in https://github.com/rakudo/rakudo/commit/f55e28b4b50c1766f42c74c34c8c7b3063d1534c